### PR TITLE
Easy open for Gamestonk Terminal following install

### PR DESCRIPTION
**Pull request by RowanHarley**
Changed readme to include optional process to speed up the terminal opening in the future, by using a simple batch file

### CODE REVIEW

**Comment by DidierRLopes** [comment]
Hey @RowanHarley , thanks for this.

What do you think about adding the batch file template as well? I think the majority of users aren't devs, and not sure they would be able to set this batch file easily.

**Comment by RowanHarley** [comment]
Good idea @DidierRLopes . Should I add it as a seperate file inside this repo?

**Comment by DidierRLopes** [comment]
> Good idea @DidierRLopes . Should I add it as a seperate file inside this repo?

Yes, I'd say so. Outside the gamestonk_terminal folder :)

File: README.md [hunk 1]
```diff
@@ -154,6 +154,9 @@ This is a library for package management, and ensures a smoother experience than
 ```
 python terminal.py
 ```
+10. (Optional) Speeding up opening process in the future
```
Comment (DidierRLopes) [inline README.md:157]: May be worth pointing out that this is for Windows only. Correct?

Comment (RowanHarley) [reply README.md:157]: Yes, good point. I'll change the readme to reflect that, and if I get the chance, I'll try to create a similar program for Mac OS to speed up the opening process

Comment (DidierRLopes) [reply README.md]: Nice sounds good!!

**Review by DidierRLopes — APPROVED** [review]

**Review by aia — APPROVED** [review]

**Pull request merged**